### PR TITLE
Pipeline2 18 create branch

### PR DESCRIPTION
**Pull request by JGuzman4**

### CODE REVIEW

**Review by jbankes — CHANGES_REQUESTED** [review]
changes requested mentioned in slack

File: libotrio.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ if (!process.env.MONGODB_URI) {
 }
 
 var controller = Botkit.slackbot({
+  clientSigningSecret: process.env.SLACK_VERIFICATION_TOKEN,
```
Comment (jbankes) [inline libotrio.js:55]: is this going to cause a problem for local debugging that we are aware of?

Comment (JGuzman4) [reply libotrio.js:55]: Not sure. I believe that variable will just be set to null like it was before

**Review by jbankes — APPROVED** [review]

**Review by jbankes — DISMISSED** [review]
Fulfilled all asks. Adds Jira ticket grabbing to Libotrio. LGTM

**Review by Nelsonater — DISMISSED** [review]
Looks good to me

**Review by Nelsonater — APPROVED** [review]
Version bump. Looks good

**Pull request merged**